.github/workflows/docker_image.yml

```diff
@@ -3,9 +3,6 @@ on:
   workflow_dispatch:
   push:
     tags: "*"
-    branches:
-      - main
-  pull_request:
```

Comment:
I have a personal preference to keep these so that experimental changes can be easily checkout out

Reply:
Ok I put them back in [f46dd70](https://github.com/Kitware/pan3d/pull/25/commits/f46dd7065b01f2d3042c5c1264f675fd80060040)

Reply:
Glad it's turned back on, because now we can see there's actually a failure. It's failing on the following command in the dockerfile:

```
COPY --chown=trame-user:trame-user ./server /deploy/server
```

Do you know what the intent of this line was? The server folder doesn't exist now.

Reply:
This is because you removed the `build.Dockerfile`. There is a specific build process for Trame. cc @jourdain

Reply:
I had thought it was an overlooked redundancy since the content of the two dockerfiles were very similar. But since it is functional, I've put it back in [2d41f8c](https://github.com/Kitware/pan3d/pull/25/commits/2d41f8c5f9f3db2330d31bfb43e012c0768ccde9). Thanks for explaining.

Reply:
@banesullivan a different error has appeared with that dockerfile put back. It looks like it's attempting to activate a venv that doesn't exist. Does this folder need to be created?

Reply:
So trame allow to build that server directory by running a command line with docker. And another one bundle that generated directory into a docker image.

You can merge the build process inside a single docker image by copying the deploy directory and do a something like that

```
RUN /opt/trame/entrypoint.sh build
```